Homework_5.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<ul><li>Reference! You have to follow the references guidelines for the wikipedia picture you use: https://commons.wikimedia.org/wiki/File:Float_example.svg</li><li>double is a complicated case, since you already figured out it is split in 3 parts, and you can technically only use the 23 bits (less than 3 bytes!) of the fractional part for this problem without loosing precision. So your calculation here is technically wrong.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>